tensorly/backend/tensorflow_backend.py

```diff
@@ -43,11 +43,11 @@ def to_numpy(tensor):
 
     @staticmethod
     def ndim(tensor):
-        return len(tensor.get_shape()._dims)
+        return len(tl.to_numpy(tensor).shape)
 
```

Comment:
For some reason, the old implementation was failing for me. This might be a problem with the dynamic dispatching though. If it turns out my implementation of something else was wrong, I'll go back and fix this.